File: 03Operadores/OperadoresLogi.java
public class OperadoresLogi
{
  public static void main(String[] args)
  {
    boolean a= true, b=false;
    System.out.println(" a && b  = " + (a && b) );
    System.out.println(" a || b  = " + (a || b) );
    //ahora usando el operador de negacion
    System.out.println(" !b  = " + (!b) );
    System.out.println(" a && !b  = " + (a && !b) );
    System.out.println(" !a || b  = " + (!a || b) );
  }
}

File: 04Oper_SecCont/OperadoresLogi.cpp
#include <iostream>
//using namespace std;

int main(){
    bool a= true, b=false;
    std::cout << " a && b = " << ( a && b ) << std::endl
        << " a || b  = "<< ( a || b ) << std::endl
        //ahora usando el operador de negacion
        << " !b  = "<< ( !b ) << std::endl
        << " a && !b = " << ( a && !b ) << std::endl
        << " !a || b  = "<< ( !a || b ) << std::endl;

    return 0;
}

